plugins/1x.js

```diff
@@ -24,6 +24,31 @@ hoverZoomPlugins.push({
             '-thumb.',
             '-fullsize.'
         );
+		hoverZoom.urlReplace(res,
+            'img[src]',
+            '-ld.',
+            '-hd2.'
+        );
+		hoverZoom.urlReplace(res,
+            'img[src]',
+            '-ld.',
+            '-sd.'
+        );
+		hoverZoom.urlReplace(res,
+            'img[src]',
+            '-sq.',
+            '-hd2.'
+        );
+		hoverZoom.urlReplace(res,
```

Comment:
This replace is not necessary. When the previous one (on line 37) completes, you'll no longer have any images with `-sq.` in them, so this replacement won't do anything.